pkg/distsql/select_result.go

```diff
@@ -547,6 +547,9 @@ func (r *selectResult) updateCopRuntimeStats(ctx context.Context, copStats *copr
 			return err
 		}
 	}
+	if copStats.TimeDetail.ProcessTime > 0 {
```

Comment:
In what situations would `ProcessTime` be zero?

Reply:
For tiflash cop response, the field will be 0 :)